build/goenv.txt

```diff
@@ -0,0 +1,7 @@
+GOCACHE=/tmp/go-cache
+GOPROXY=file:///mnt/goproxy
```

Comment:
Are these still necessary with us not using volume mounts?

Reply:
I have no clue. i just copied the environment from the original build process 🙂

Reply:
The answer is no